src/interface/swagger.json

```diff
@@ -2114,52 +2250,60 @@
         },
         "QuotesIDPutResponse": {
             "title": "QuotesIDPutResponse",
-            "description": "PUT /quotes/{ID} object",
+            "type": "object",
+            "description": "The object sent in the PUT /quotes/{ID} callback.",
             "required": [
                 "transferAmount",
                 "expiration",
                 "ilpPacket",
                 "condition"
             ],
-            "type": "object",
             "properties": {
                 "transferAmount": {
-                    "$ref": "#/definitions/Money",
-                    "description": "The amount of money that the Payee FSP should receive."
+                    "description": "The amount of money that the Payee FSP should receive.",
+                    "type": "object",
+                    "$ref": "#/definitions/Money"
                 },
                 "payeeReceiveAmount": {
-                    "$ref": "#/definitions/Money",
-                    "description": "The amount of Money that the Payee should receive in the end-to-end transaction. Optional as the Payee FSP might not want to disclose any optional Payee fees."
+                    "description": "The amount of Money that the Payee should receive in the end-to-end transaction. Optional as the Payee FSP might not want to disclose any optional Payee fees.",
+                    "type": "object",
+                    "$ref": "#/definitions/Money"
                 },
                 "payeeFspFee": {
-                    "$ref": "#/definitions/Money",
-                    "description": "Payee FSP’s part of the transaction fee."
+                    "description": "Payee FSP’s part of the transaction fee.",
+                    "type": "object",
```

Comment:
i dont think type is required as the "Money" definition include type

Reply:
fixed